package.json

```diff
@@ -17,9 +17,9 @@
     "prebuild": "npm run format && npm run lint",
     "build": "rm -rf dist && tsc",
     "postbuild": "shx chmod +x dist/*.js",
-    "dev": "tsx . run",
-    "dev:debug": "DEBUG=auth0-mcp tsx . run",
-    "dev:inspect": "npx @modelcontextprotocol/inspector tsx . run",
+    "dev": "node --import tsx src/index.ts run",
+    "dev:debug": "DEBUG=auth0-mcp node --import tsx src/index.ts run",
+    "dev:inspect": "npx @modelcontextprotocol/inspector node --import tsx src/index.ts run",
```

Comment:
Use Node's native ESM import feature for TypeScript execution. This eliminates the need to run `npm run build` before `npm run dev`, allowing developers to see changes immediately when modifying TypeScript files.